src/python/pants/option/global_options.py

```diff
@@ -327,6 +327,14 @@ def register_bootstrap_options(cls, register):
              help='Paths that correspond with build roots for any subproject that this '
                   'project depends on.')
     register('--owner-of', type=list, member_type=file_option, default=[], daemon=False, metavar='<path>',
+             removal_version="1.27.0.dev0",
+             removal_hint=(
+               "Use direct file arguments instead, such as "
+               "`./pants list src/python/f1.py src/python/f2.py` or even "
+               "`./pants fmt 'src/python/**/*.py'`. Instead of `--owner-of=@my_file`, use "
```

Comment:
```suggestion
               "`./pants fmt 'src/python/**/*.py'`.\nInstead of `--owner-of=@my_file`, use "
```
The continuity made it confusing for me. For a moment, I thought that --spec-file was a replacement for `--owner-of=a.txt`, not `--owner-of=@a.txt`

Reply:
Great suggestion. Thanks!